rust/agents/relayer/src/settings/whitelist.rs

```diff
@@ -0,0 +1,209 @@
+use std::fmt;
+use std::fmt::Formatter;
+use std::marker::PhantomData;
+use std::num::ParseIntError;
+
+use ethers::types::H256;
+use serde::de::{Error, SeqAccess, Visitor};
+use serde::{Deserialize, Deserializer};
+
+use abacus_core::AbacusMessage;
+
+/// Whitelist defining which messages should be relayed. If no wishlist is provided ALL
+/// messages will be relayed.
+///
+/// Valid options for each of the tuple elements are
```

Comment:
Wondering if these comments here are more appropriate for WhitelistElement

Reply:
Whitelist element is a private struct that no one should ever directly see. I treat it as a "dumb C-style struct".